01_install_requirements.sh

```diff
@@ -59,3 +59,21 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
   -e "virthost=$HOSTNAME" \
   -i vm-setup/inventory.ini \
   -b -vvv vm-setup/install-package-playbook.yml
+
+
+if [ ! -d "/etc/containers" ] ; then
```

Comment:
What is the `/etc/containers` config needed for?  Can the Dockerfiles just be explicit about which registry to pull from?